pyproject.toml

```diff
@@ -78,6 +75,13 @@ sphinx-autodoc-typehints = "^1.12.0"
 sphinx-rtd-theme = "^0.5.2"
 types-PyYAML = "^6.0.4"
 
+[tool.poetry.group.extra]
+optional = true
+
+[tool.poetry.group.extra.dependencies]
```

Comment:
let's call this group `featurestore` extra is reserved for python/pypi stuff

Reply:
Good idea!